Loop/Managers/Alerts/AlertManager.swift

```diff
@@ -106,19 +106,23 @@ extension AlertManager: AlertManagerResponder {
     }
     
     func presentAcknowledgementFailedAlert(error: Error) {
-        let message: String
-        if let localizedError = error as? LocalizedError {
-            message = [localizedError.localizedDescription, localizedError.recoverySuggestion].compactMap({$0}).joined(separator: "\n\n")
-        } else {
-            message = String(format: NSLocalizedString("%1$@ is unable to clear the alert from your device", comment: "Message for alert shown when alert acknowledgement fails for a device, and the device does not provide a LocalizedError. (1: app name)"), Bundle.main.bundleDisplayName)
```

Comment:
I suggest we remove `presentAcknowledgementFailedAlert` and rename `acknowledgeAlert` -> `alertAcknowledged`